### DISCUSSION TITLE
TV and PC setups - thoughts and questions

### DISCUSSION THREAD

**Discussion by brh150** [awawa-dev/HyperHDR#645] [General]
Hello everyone.

I'd like to make two builds - one for a TV (42 inches) + chromecast 4k and one for a PC (27 inches monitor). The TV is wall-hung above my PC desk on which also stands PC case.

What I have:
- LEDs - 5m of SK6812 RGB NW 60 LEDs/1m. ~3m should do for the TV and <2m for the monitor;
- Capture card - MS2130 USB 3.0.

What I am thinking to buy:
- HDMI splitter - Ezcoo EZ-SP12HAS (~32,5 euros);
- LED drivers - Feather RP2040 SCORPIO (~17,5-19,5 euros) for the TV, Adafruit ItsyBitsy RP2040 (~15 euros) for the monitor or Pimoroni Plasma 2040 (~20 euros) x2 just for the sake of easy connection (screw terminals);
- PSU - Mean Well LRS-150F-5 - 5V/22A/110W (~23 euros) or POS Power POS-150-5-C - 5V/22A/110W (~17,5 euros).

Some thoughts and questions:
1. HyperHDR is going to run on my PC as it is usually turned on when I want to watch a movie on the TV in the evening.
2. Splitter with CEC support to retain TV controll with cromecast's remote (do I understand CEC support correctly?).
3. RP2040 SCORPIO for the TV is probably overkill, but should be futureproof if I change the TV for something bigger and maybe make use of multi-segment connection. For current TV (42 inches) and ~170 LEDs, however, single segment should be OK performance wise from what I have read. Plasma 2040 could be an alternative.
4. ItsyBitsy RP2040 for the monitor as single-segment should be enough for ~110 LEDs performance wise and it is a little bit cheaper than SCORPIO. Plasma 2040 could be an alternative.
5. I was thinking to power both LED instances with one PSU. Those SK6812 LEDs are rated ~4A per 1m. Mean Well LRS-150F-5 and POS 150-5-C should be theoretically capable of powering them simultaneously (up to 5m in total = up to 300 LEDs = ~20A). From what I have read Mean Wells are tried and tested. Don't know about POS. It's cheaper, that's for sure. What do you think?
6. Hardware management - location of LED drivers and other stuff. Where to put them? Does it even matter cable length wise - USB cables, power and data cables etc? PSU will go into a box, that's for sure. What about other stuff?
Other builds I've seen usually used RPi and everything was mounted behind a TV or hidden in a box and put close to a TV. 
8. One thing is bugging me about LED drivers - is it even possible to use just one with multiple line support like SCORPIO to run several independent instances - TV and PC monitor in my case? Couldn't find such information, so I guess not, but I'd like to be sure.
9. Anything else I should think of before I jump to make thees builds?

Your input is much appreciated!

**Comment by AstaRom** [comment]
> 2\. Splitter with CEC support to retain TV controll with cromecast's remote (do I understand CEC support correctly?

The above splitter supports CEC.
I use this one myself.



> 5\. From what I have read Mean Wells are tried and tested. Don't know about POS. It's cheaper, that's for sure. What do you think?

Mean Well power supplies are very good. It’s not for nothing that they are used in industrial equipment (for example, in our server room these units have been working without any problems for probably 9 years without a break).

**Comment by awawa-dev** [comment]
As @AstaRom mentioned, the cable length from the LED driver to the LED strip could be an issue: the signal is prone to interference  (and voltage drop especially if the level shifter is not used but that's not the case here). You could use up to two segments of different length if 'the second segment reversed' option is not used. Configuring its geometry (one combined LED strip for two monitors) in HyperHDR is another subject, didn't try it.

**Reply by AstaRom** [reply]
> but that's not the case here

Indeed, I wrote about the wrong data line. In my head I had an image of the SPI bus from the driver to the RPi (which should be as short as possible as you said earlier in other topics). Here we were talking about the data line from the driver to the LED strip 😀

**Comment by brh150** [comment]
@AstaRom, @awawa-dev, thank you both for your input and sorry for my late response.

@awawa-dev, I saw your build - nicely done.

In the meantime I've bought Ezcoo EZ-SP12HAS. For LED drivers I've decided to go with two Pimoroni Plasma 2040 boards - it took some time for a shop to restock them, so I have bought it just few days ago. And as for the PSU - I've found quite a good deal on a used Mean Well LRS-150F-5 - half price of a new one. According to the seller it was used no more than 60 hours. I still have to buy some things like electrical box, cables, power switches etc. I'm not in a hurry, so maybe I'll manage to finish it before New Year - will see :)

Thank you again.
